src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -4788,6 +4789,8 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						}
 					} else if (!strcasecmp(var, "odbc-dsn") && !zstr(val)) {
 						profile->odbc_dsn = switch_core_strdup(profile->pool, val);
+					} else if (!strcasecmp(var, "odbc-start-delay") && !zstr(val)) {
```

Comment:
Please edit vanila conf putting that param in a commented out form. Give it a comment about defaults.